examples/scripts/ppo/ppo_tldr.py

```diff
@@ -116,15 +116,15 @@ def tokenize(element):
         train_dataset = prepare_dataset(train_dataset, tokenizer)
         eval_dataset = prepare_dataset(eval_dataset, tokenizer)
         # filtering
-        train_dataset = train_dataset.filter(lambda x: x["lengths"] <= 512, num_proc=config.dataset_num_proc)
-        eval_dataset = eval_dataset.filter(lambda x: x["lengths"] <= 512, num_proc=config.dataset_num_proc)
+        train_dataset = train_dataset.filter(lambda x: x["lengths"] <= 512, num_proc=training_args.dataset_num_proc)
+        eval_dataset = eval_dataset.filter(lambda x: x["lengths"] <= 512, num_proc=training_args.dataset_num_proc)
 
     assert train_dataset[0]["input_ids"][-1] != tokenizer.eos_token_id, "The last token should not be an EOS token"
     ################
     # Training
     ################
     trainer = PPOv2Trainer(
-        config=config,
+        config=training_args,
```

Comment:
Not for this PR, but is there a reason the PPO/RLOO configs have a `config` attribute instead of the default `args` one?

Reply:
I don't think so. I think we just didn't considered consistency when it was implemented.